images.md

```diff
@@ -0,0 +1,22 @@
+# Lambda Instance images creation
+
+## Description
+
+Using the fokia library, an administrator is able to create updated images for the Lambda Instance Master and Slave nodes.
+
+## Usage
+
+1. To create the Lambda Instance images, one must use the fokia library. This can be done either on your own computer, after installing the required dependencies, or using an up-to-date Lambda Service VM.
+2. Navigate to the fokia library folder: `cd <project_directory>/core/fokia`.
+3. Run `python lambda_instance_manager.py --image-creation --project-name <~okeanos project name> --auth_token <~okeanos auth token>`, to create a minimal Lambda Instance suitable for image creation.
```

Comment:
No, the name of the image is defined in the snf-image-creator documentation. I think that to create an image, one has to read the snf-image-creator documentation which contains all of the above, and understand how to use it.